ansible/roles/test/files/tools/loganalyzer/loganalyzer_end.yml

```diff
@@ -17,11 +17,11 @@
   register: errors_found
 
 - name: Check if loganalyzer missed expected messages
-  shell: grep "TOTAL EXPECTED MATCHES" "{{ test_out_dir }}/{{ summary_file }}" | sed -n "s/TOTAL EXPECTED MATCHES:[[:space:]]*//p"
-  register: expected_matches
+  shell: grep "TOTAL EXPECTED MISSING MATCHES" "{{ test_out_dir }}/{{ summary_file }}" | sed -n "s/TOTAL EXPECTED MISSING MATCHES:[[:space:]]*//p"
+  register: expected_missing_matches
 
 - set_fact:
-    fail_in_logs: "{{ errors_found.stdout  != \"0\" or expected_matches.stdout != \"0\" }}" 
+    fail_in_logs: errors_found.stdout != "0" or expected_missing_matches.stdout != "0"
```

Comment:
I think without the quote, it failed comparison while I tried it. Did you check this in your running? what's your ansible version?

Reply:
2.0.0.2
I reverted this change.